apps/meteor/client/views/omnichannel/triggers/actions/ActionExternalServiceUrl.tsx

```diff
@@ -0,0 +1,99 @@
+import { Box, Button, Field, FieldError, FieldHint, FieldLabel, FieldRow, Icon, Palette, TextInput } from '@rocket.chat/fuselage';
+import { useUniqueId } from '@rocket.chat/fuselage-hooks';
+import type { TranslationKey } from '@rocket.chat/ui-contexts';
+import { useEndpoint, useTranslation } from '@rocket.chat/ui-contexts';
+import { useMutation } from '@tanstack/react-query';
+import type { ComponentProps } from 'react';
+import React, { useState } from 'react';
+import type { Control, UseFormTrigger } from 'react-hook-form';
+import { Controller, useWatch } from 'react-hook-form';
+
+import type { TriggersPayload } from '../EditTrigger';
+import { useFieldError } from '../hooks/useFieldError';
+
+type ActionExternaServicelUrlType = ComponentProps<typeof Field> & {
+	index: number;
+	control: Control<TriggersPayload>;
+	trigger: UseFormTrigger<TriggersPayload>;
+	disabled?: boolean;
+};
+
+const SUCCESS_COLOR = Palette.statusColor['status-font-on-success'].toString();
+
+export const ActionExternalServiceUrl = ({ control, trigger, index, disabled, ...props }: ActionExternaServicelUrlType) => {
+	const t = useTranslation();
+
+	const serviceUrlFieldId = useUniqueId();
+	const serviceUrlFieldName = `actions.${index}.params.serviceUrl` as const;
+	const serviceTimeoutFieldName = `actions.${index}.params.serviceTimeout` as const;
+
+	const serviceTimeoutValue = useWatch({ control, name: serviceTimeoutFieldName });
+	const [serviceUrlError] = useFieldError({ control, name: serviceUrlFieldName });
+	const [isSuccessMessageVisible, setSuccessMessageVisible] = useState(false);
```

Comment:
const isSuccessMessageVisible = useDebouncedValue(webhookTest.isLoading, 10000);

Reply:
I tried a few variations but unfortunately this did not work as the scenario here is a bit different than the one we saw together.